lcd/config_downloader.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import uuid
+import requests
+
+class ConfigDownloader:
+    def __init__(self, config_path, remote_url):
+        self.config_path = config_path
+        self.remote_url  = remote_url
+        self.api_key     = uuid.uuid4()
+        self.error       = None
+        self.response    = None
+
+    def run(self):
+        print "Setting up a new account"
+        print "please make sure an account with API key %s is configured" % self.api_key
+        ready = raw_input("press return when ready... ")
+        if ready != "":
+            print "aborting..."
+            exit()
+        if self.download():
+            print "writing config file to: " + self.config_path
+            self.save()
+            print "done."
+        else:
+            print "dowload failed:"
+            print self.response
+            print self.error
+
+    def download(self):
+        try:
+          self.response = requests.get(self.remote_url, headers={'X-Api-Key': self.api_key})
+          return self.response.status_code == requests.codes.ok
+        except Exception as e:
+          self.error = e
+          return False
+
+    def save(self):
+        f = open(self.config_path,"w")
```

Comment:
done. simply catching everything. 

I would not use a default path as the ConfigDownloader requires a config_path to be actively specified by whoever uses the module